hyp3_gamma/__main__.py

```diff
@@ -126,6 +137,7 @@ def insar():
     parser.add_argument('--include-inc-map', type=string_is_true, default=False)
     parser.add_argument('--apply-water-mask', type=string_is_true, default=False)
     parser.add_argument('--looks', choices=['20x4', '10x2'], default='20x4')
+    parser.add_argument('--phase-filter-threshold', type=valid_range_float, default=0.6)
```

Comment:
It'd be worth checking the GAMMA docs to see what they call this parameter.

Reply:
I will check the width in the GAMMA.

Reply:
Looking at the GAMMA docs, this is the exponent in the phase filter. I'm unsure what the valid range is now, but the algorithm is described in this paper:
https://doi.org/10.1029/1998GL900033